google/cloud/aiplatform/vizier/trial.py

```diff
@@ -146,8 +163,20 @@ def complete(
             else None
         )
 
-    def should_stop(self) -> bool:
-        """Returns true if the Trial should stop."""
+    def check_early_stopping(self) -> bool:
```

Comment:
same comment as above. this is a breaking change

Reply:
Looks like the Vertex Vizier SDK is not used too much by the customers. I think it's fine to make the change on the API ? https://datastudio.google.com/c/u/0/reporting/d8a64931-8a91-4f07-9abc-93fece2e318d/page/p_n2odnmsgmc